README.md

```diff
@@ -63,6 +63,7 @@ You can customize all of the deployment parameters of the LSP simply by changing
 --basePercentage: The percentage of collateral per pair used as the floor. This parameter is used with the 'SuccessToken' fpl where the remaining percentage functions like an embedded call option.
 --lowerBound: Lower bound of a price range for certain financial product libraries. Cannot be included if --strikePrice is specified.
 --upperBound: Upper bound of a price range for certain financial product libraries.
+--simulate: Boolean telling if set the script should only simulate the transactions without sending them to the network.
```

Comment:
nit: `Boolean telling if the script should only simulate the transactions without sending them to the network.`